packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -121,6 +121,26 @@ if (!ExecutionEnvironment.canUseDOM) {
     },
   };
 
+  /**
+   * Handles the case where a callback errors:
+   * - don't catch the error, because this changes debugging behavior
+   * - do start a new postMessage callback, to call any remaining callbacks,
+   * - but only if there is an error, so there is not extra overhead.
+   */
+  const callSafely = function(callback, arg) {
```

Comment:
`callSafely` implies that the function won't throw. Can we rename this to `callUnsafely` or similar?